package.json

```diff
@@ -33,28 +19,25 @@
     "babel-preset-react": "^6.11.1",
     "babel-register": "^6.16.3",
     "babelify": "^7.3.0",
-    "body-parser": "~1.15.1",
-    "cf-deployment-tracker-client": "0.1.1",
-    "cookie-parser": "^1.4.0",
-    "csurf": "^1.9.0",
+    "body-parser": "^1.15.2",
+    "cf-deployment-tracker-client": "^0.1.1",
     "dotenv": "^2.0.0",
-    "eslint-plugin-react": "^6.3.0",
-    "express": "~4.13.4",
+    "express": "^4.14.0",
     "express-browserify": "^1.0.2",
-    "express-rate-limit": "^2.3.1",
+    "express-rate-limit": "^2.5.0",
     "express-react-views": "^0.10.2",
     "express-secure-only": "^0.2.1",
     "find-remove": "^0.2.8",
-    "helmet": "^2.1.1",
+    "helmet": "^2.3.0",
     "morgan": "^1.7.0",
-    "multer": "^1.1.0",
+    "multer": "^1.2.0",
     "prismjs": "^1.5.1",
     "react": "^15.3.1",
     "react-dom": "^15.3.2",
     "react-prism": "^3.2.2",
     "uglifyify": "^3.0.3",
     "watson-developer-cloud": "^2.2.0",
-    "watson-react-components": "0.0.16",
+    "watson-react-components": "^0.0.16",
```

Comment:
Isn't this the same thing? :P

Relatedly, we should probably ship a minor release so that we can at least differentiate between bug fixes and more significant changes. But, meh.